packages/client/modules/meeting/components/AddNewReflectTemplate.tsx

```diff
@@ -28,13 +30,43 @@ const AddRetroTemplateLink = styled(LinkButton)({
 
 interface Props {
   gotoTeamTemplates: () => void
-  reflectTemplates: AddNewReflectTemplate_reflectTemplates
-  teamId: string
+  reflectTemplatesRef: AddNewReflectTemplate_reflectTemplates$key
+  viewerRef: AddNewReflectTemplate_viewer$key
+  setShowUpgradeDetails: (showUpgradeDetails: boolean) => void
```

Comment:
-1 it's only ever called with `true` so it should be `showUpgradeDetails: () => void` instead